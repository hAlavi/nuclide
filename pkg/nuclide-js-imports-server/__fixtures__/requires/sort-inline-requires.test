const {Y} = require('X');
const X = require('X');

const x = X + Y;
