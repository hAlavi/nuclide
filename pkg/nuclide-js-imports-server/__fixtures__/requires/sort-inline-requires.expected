const X = require('X');
const {Y} = require('X');

const x = X + Y;
